tools/api-markdown-documenter/src/test/snapshots/markdown/simple-suite-test/deep-config/test-suite-a/testclass-class/testclassgetterproperty-property.md

```diff
@@ -9,6 +9,7 @@ Test class property with both a getter and a setter.
 ```typescript
 /** @virtual */
 get testClassGetterProperty(): number;
+
```

Comment:
Note for reviewers: this is actually correct. The previous rendering was incorrect. The JSON that API-Extractor generated, which this is derived from, is:
```json
"excerptTokens": [
		{
			"kind": "Content",
			"text": "get testClassGetterProperty(): "
		},
		{
			"kind": "Content",
			"text": "number"
		},
		{
			"kind": "Content",
			"text": ";\n\nset testClassGetterProperty(newValue: number);"
		}
	],
```